<html>
<head>
<title>About SimPol</title>
<link rel="icon" href="../src/Images/transcribe.png">
<meta name="viewport" content="width=device-width, initial-scale=0.41, maximum-scale=1" />
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="../src/Resources/gifplayer-master/dist/gifplayer.css">
<link rel="stylesheet" type="text/css" href="../src/Resources/styles.css">
<script language='JavaScript' type='text/JavaScript' src="../src/Resources/jquery-3.1.1.min.js"></script>
<script language='JavaScript' type='text/JavaScript' src="../src/Resources/bibliography.js"></script>
<script type="text/javascript" src="../src/Resources/gifplayer-master/dist/jquery.gifplayer.js"></script>

<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-106202043-1', 'auto');
  ga('send', 'pageview');

</script>


<script language='JavaScript' type='text/JavaScript'>


function init(){

	IS_MOBILE = false; 
	IS_MOBILE = (/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|ipad|iris|kindle|Android|Silk|lge |maemo|midp|mmp|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows (ce|phone)|xda|xiino/i.test(navigator.userAgent) 
		 || /1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(navigator.userAgent.substr(0,4)));

	setImageSizes();
	numReferences = 0;
	referenceNumbers = {};
	addCitations();
	
	$('.gif').gifplayer();


	// If the url is linked to a section then make sure the section is not collapsed
	// Sections given id in this format: RateBind_ParamHelp. This means the linkable section is in the Parameters section
	if(window.location.hash) {
		//console.log(window.location.hash);

		var hashBits = window.location.hash.split("_");
		if (hashBits[1] == "ParamHelp") minimiseSection("#parametersBtn", true);
		else if (hashBits[1] == "PlotHelp") minimiseSection("#displayingPlotsBtn", true); 
		else if (hashBits[1] == "ModelHelp") minimiseSection("#kineticsBtn", true); 
		else if (hashBits[1] == "ABCSectionHelp") minimiseSection("#ABCSectionBtn", true); 




		$(window.location.hash).addClass("helpSelected");
		
		
		// Add back the anchor to ensure that the page is in the right place. If the webpage takes too long to load and
		// the anchored section is hidden, then the page won't anchor
		var hash = window.location.hash;
		window.location.hash = hash;
		window.location.hash = window.location.hash;


	}
	
}


function setImageSizes(){


									
	if(IS_MOBILE){

		$(".main-block").css("width", "90%");

		$("#polStructure").width("80%");
		$("#polStructure").css("height", "");
		$("#structureParams").width("80%");
		$("#structureParams").css("height", "");
		$("#breakingPairs").width("80%");
		$("#breakingPairs").css("height", "");
		$("#exponential").width("80%");
		$("#exponential").css("height", "");

	}



}


function minimiseSection(element, setTo = null){

	var toHide = $(element).attr("id");
	toHide = toHide.substring(0, toHide.length-3); // Remove 'Btn' from the end
	console.log(toHide);

	
	if(setTo == false || (setTo == null && $(element).attr("disp") == "false")){
		$(element).html("&plus;")
		$("#" + toHide).hide(setTo == null ? 300 : 0);
		$(element).attr("disp", true);
	}
	else{
		$(element).html("&minus;")
		$("#" + toHide).show(setTo == null ? 300 : 0);
		$(element).attr("disp", false);
	}

}

function openReferences(){

	
	minimiseSection("#referencesSectionBtn", true);

}

$(window).on('hashchange', function() {
  	$(window.location.hash).addClass("helpSelected");
});


function addCitations(){


	var citationTags = $("cite");
	for (var i = 0; i < citationTags.length; i ++){
	
		
		var vals = $(citationTags[i]).attr("val").split(",");
		if (vals.length == 0) continue;
		
		var citationHTML = "<sup class='refLink'>(";
		
		for (var j = 0; j < vals.length; j ++){
		
			// Add the reference to the bottom of the page if it does not already exist
			var val = vals[j].trim();
			if ($("#" + val).length == 0){
			
				numReferences++;
				referenceNumbers[val] = numReferences;
				var referenceString = BIBLIOGRAPHY[val];
				$("#references").append('<div class="ref" id="' + val + '">' + numReferences + ') ' +  referenceString + '</div><br>');

			}
			
			citationHTML +=  `<a onclick="openReferences();" class='refLink'  href="#` + val + `">` + referenceNumbers[val] + `</a>`;
			if (j < vals.length-1) citationHTML += ",";
			
		}
		
		
		// Create the link to the reference
		citationHTML += ")</sup>";
		$(citationTags[i]).append(citationHTML);
		

	
	}



};



</script>

</head>

<body onload="init()">
	<div id="screenContainer" style="position:relative">
		<div id="screen" style="opacity:1;">


				

			<div id="main" style="background-color:white">
				
				
				<div id="header" class="header" style="width:100%; ">
					<table style="margin: 0px auto; width:100%">
						

						
						<tr>
							<td style="height:30px; width: 30%; vertical-align:bottom; ">
								
								
							</td>

							<td  style="text-align:center; font-size: 20px; vertical-align:top; font-family: 'Palatino'; width: 40%;"> 
								<div style="width:100%; text-align:center">
									<a href="../" style="text-decoration: none; vertical-align:middle; width:300px">
										<img src="../src/Images/logo.png" style="height:110px; vertical-align:middle;">
									</a>
									
									
									
									
									
								</div>
										
							</td>
							
							

							<td style="height:30px; width: 30%; vertical-align:bottom; text-align:right;"> 
								<span class="noselect" style="padding:5 5; background-color:#008CBA; padding: 5 5; color:white;"><a href="../" style="text-decoration:none; color:white; ">Home</a></span> &nbsp;
								<span class="noselect" style="padding:5 5; background-color:#b20000; padding: 5 5; color:white;">About</span> &nbsp;
								<span class="noselect" style="padding:5 5; background-color:#008CBA; padding: 5 5; color:white;"><a href="../download" style="text-decoration:none; color:white; ">Download</a></span> &nbsp;
								<span class="noselect" style="padding:5 5; background-color:#008CBA; padding: 5 5; color:white;"><a href="../contact" style="text-decoration:none; color:white; ">Contact us</a></span> &nbsp;
							</td>
						</tr>

						
					</table>
				</div>
			   
				
				

				<br>
				<div class="about-block" style="width:70%; margin:auto">
					<div class="about-block-inner">
						<div class="block-subtitle" style="font-size:45px"> An interactive graphical simulation of nucleic acid polymerases </div> <br>
					
						<b>Disclaimer:</b> SimPol is currently in prerelease. It has not been peer-reviewed or published. <br><br>
					
						SimPol is a free open-source web-based program aimed to help researchers, teachers and students visualise transcription at the level of a single molecule. Transcription is the copying of a template (typically DNA) 
						into messenger RNA (mRNA) and is one of the most central processes in all of biology. The copying process is carried out by a protein called RNA polymerase. RNA polymerase runs along the template 
						as it extends the mRNA. While this main pathway of transcription is fairly straightfoward, some of the less-common and off-pathway events can be	cause complications. For example, sometimes	
						RNA polymerase moves very far in the wrong direction (backtracking), sometimes a nucleotide is copied incorrectly (a misincorporation), and sometimes the template and mRNA slip along each other (which can cause an insertion or deletion).
						Researchers have built kinetic models of transcription to describe and quantify the main pathway and some of these off-pathway processes.
						<br><br>

						
						This program has two main functions: (1) <b>Visualisation</b> of transcription elongation, misincorporations and transcriptional slippage, and for more advanced users (2) <b>Simulation</b> of transcription elongation  
						with kinetic models. This webpage contains tutorials on how to use SimPol.   <br><br>
						
						
						
						<!--<img class="gif"  src="../src/TutorialImages/overview.png" style="width:100%; display:inline-block; margin:auto; padding: 10 10">-->

						

						
				
					</div>
				</div>
				
				
				
				<br><br><br>
				<div class="main-block" >

					<div class="main-block-inner">
						<div class="block-title"> <span id='gettingStartedBtn' class='minimise' style="font-size:30px; padding: 5 5" disp="false" onClick="minimiseSection(this)">&minus;</span> Getting Started </div> 
						<div id="gettingStarted">

							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div id="sequenceSelection" class="block-subtitle"> Sequence selection </div> <br>
									
									
									<img src="../src/TutorialImages/sequenceParams.png" style="width:300px; padding: 10 30;" align=left></img>
									
									<br><br><br>
									
									The first thing you may want to do is choose your nucleic acid sequence. Open up the &#9776; Parameters side-navigation menu, where you can enter your own sequence or 
									use one of the example sequences.
									
									
									
									SimPol is designed primarily for transcription with DNA-dependent RNA-polymerases. Your choice of template-type (DNA or RNA) and nascent-type (DNA or RNA) will dictate
									which thermodynamic parameters are used - Turner's RNA/RNA parameters<cite val="xia1998thermodynamic"></cite>, Sugimoto's DNA/RNA parameters<cite val="wu2002temperature"></cite>, or SantaLucia's DNA/DNA parameters<cite val="santalucia1998unified"></cite>. <br><br>

								


							
								</div>
							</div>



				
							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div class="block-subtitle"> Polymerase structure </div> <br>
								


									Polymerases read the template strand (eg. the genome) in the 3&prime; &rarr; 5&prime; direction, and copy it to create the nascent strand (eg. mRNA) which grows in the 5&prime; &rarr; 3&prime; direction. You can control the polymerase using the Navigation panel
									(described in the section below). RNA polymerases typically contain four channels: a template entry channel, a template exit channel, a nascent strand exit channel and an NTP entry pore. Within the RNA polymerase, 
									the template and nascent strand basepair to form a hybrid. If the template is double-stranded then RNA polymerase needs to melt the template in order to access the strand which is being copied. 
									This set of bases which are melted comprises the transcription bubble. In the example below, the transcription bubble contains 2 + 9 + 1 = 12bp.  <br><br>
									

									<img id="polStructure" src="../src/TutorialImages/polStructure.png" style="height:200px; padding: 10 30;" align=left></img>
									<img id="structureParams" src="../src/TutorialImages/structureParams.png" style="height:200px; padding: 10 30;" align=right></img>
									
									 <br><br><br><br><br><br><br><br><br><br><br><br><br>
									The number of template bases within the transcription bubble differs between polymerases.
									These three parameters may have an effect on the activity of the polymerase and can be changed from the &#9776; Parameters side-navigation menu. 

							
								</div>
							</div>


							<br><br>
							<div class="about-block">
								<div  class="about-block-inner">
									<div id="navigationPanelHelp" class="block-subtitle" > Applying chemical reactions </div> <br>
								

									
									You can control the polymerase directly using the Navigation panel at the top of the main page. You should play around with the different reactions to see how they work. You may wish to
									Simulate <img src="../src/Images/dice.png"  style="vertical-align:middle; width:25px; height:25px;"></img> reactions, whereby the reactions described below are selected randomly based off experimental parameters.
									
								
									<br><br>
									<span class="block-subsubtitle" > Translocation. </span> There are two translocation reactions: Backwards and Forwards. These operations move the polymerase one base-pair left and right respectively, with respect to the 
									template and nascent strand. These operations involve reconfiguring the base-pairs within the hybrid, and within the chromosome if it is double-stranded. If the polymerase moves beyond the 3&prime; end of the nascent strand, 
									then the nascent strand will be released (termination). 
									

									<br><br><br>
									<span class="block-subsubtitle" > Catalysis. </span> 
									
										If the active site is free and next to the 3&prime; end of the nascent strand, then the polymerase may bind NTP.
										Once bound, the polymerase may catalyse the reaction, or it may release the NTP back into the cell. Reverse-catalysis is also possible in SimPol using 
										the Decay reaction, although it is very improbable in nature. The canonical pathway of transcription 
										elongation is a three step cycle: 1) forward translocation, 2) NTP binding, 3) catalysis. This may also be achieved with the Copy 
										<img src="../src/Images/transcribe.png"  style="vertical-align:middle; width:25px; height:25px;"></img> function. 
									</span>
									
									
									<br><br><br>
									<span class="block-subsubtitle" > Deactivation. </span>  RNA polymerases are believed to occassionally deactivate whereby they undergo 
									conformational changes and enter a catalytically inactive state, sometimes known as the intermediate state. In this state the protein cannot polymerise but it may still translocate. Factors known to increase the 
									rate of entering the inactive state are 1) weak basepairing in the hybrid, 2) interactions between the polymerase and upstream or downstream elements of the template, and 3) interactions between the polymerase and
									a secondary structure in the nascent strand <cite val="artsimovitch2000pausing,depken2009origin,imashimizu2013intrinsic"></cite>. Inactivation causes the polymerase to pause indefinitely, 
									and elongation can only resume following an activation event.
									
									
									<br><br><br>
									<span class="block-subsubtitle" > Slippage. </span> Sometimes the nascent and template strand temporarily melt and a bulge forms in one of the strands. This can be performed by the
									Form Bulge reaction.
									When this happens, a bulge (kink) forms within the duplex. This can occur within either the template or nascent strand (although in SimPol it can currently only occur within the nascent strand). When a bulge forms,
									the active site may be opened up, allowing one of the template bases to be copied twice. This would lead to an insertion. The bulge may 
									diffuse left or right along the
									 hybrid. This process is favoured if the hybrid sequence is repetitive (eg. AAAAAAAAA/UUUUUUUUU). When positioned at either end of the hybrid, the bulge may be absorbed. This three step process (bulge formation, diffusion and absorption)
									 was inspired by Neher et al.<cite val="neher2004dynamics,kuhner2007force"></cite> and is believed to be the mechanism behind transcriptional slippage. Iterative slippage and elongation can lead to large insertions. This is sometimes called stuttering and
									can also be achieved with the Stutter <img src="../src/Images/stutter.png"  style="vertical-align:middle; width:25px; height:25px;"></img> function.
							
								</div>
							</div>




							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div id="toolbarHelp" class="block-subtitle"> The toolbar </div> <br>


									<table class="toolbar" cellpadding=3 style="position:absolute; vertical-align:middle">
										
										<tr>
											<td>
												<div class="toolbarIcon" title="Reinitiate the polymerase"> <img src="../src/Images/reset.png" style=" vertical-align: middle; padding:0 0; width:35px; height:35px"> </div>
											</td>


											<td>
												<div class="toolbarIcon simulateRow" title="Perform N simulations stochastically"> <img src="../src/Images/dice.png" style="vertical-align: middle; width:35px; height:35px;">   </div>
											</td>


										</tr>
										<tr>

											

											<td>
												<div class="toolbarIcon" title="Clear data from the cache" > 
													<img src="../src/Images/close.png" style="padding:5 0; height:25px;">
												</div>
											</td>


											<td>
												<div class="toolbarIcon" title="Replicate/transcribe N bases"> <img id='trans'  src="../src/Images/transcribe.png" style="vertical-align: middle; width:35px; height:35px;">  </div>
											
											</td>

										</tr>
										<tr>

											<td>
												<div id="foldBtnDiv" class="toolbarIcon"  title="Fold the mRNA"> <img  src="../src/Images/folding.png" style="vertical-align: middle; width:35px; height:35px;" >  </input> </div>
											</td>

											<td>
												<div id="stutterBtn" class="toolbarIcon" title="Stutter N times" > <img src="../src/Images/stutter.png" style="vertical-align: middle; width:35px; height:35px;">  </div>
											
											</td>


										<tr>
											<td colspan=2>
												<div class="toolbarIcon" style="width:90px"> <span style="font-size: 16px; display:none; cursor: auto"></span> <span id="numSimSpan"> <b>N=</b><input type="number" name="nbasesToSimulate" id="nbasesToSimulate" value=30 style="vertical-align: middle; text-align:left; width: 60px; background-color: #008CBA; color:white"> </span> </div>
											</td>

										</tr>
										
										


										<tr>
											<td colspan=2>
												<div class="toolbarIcon">
													<select class="dropdown" title="Duration of each reaction" style="vertical-align: middle; text-align:right;">
														<option value="slow">Slow</option>
														<option value="medium">Medium</option>
														<option value="fast">Fast</option>
														<option value="ultrafast">Ultrafast</option>
														<option value="hidden">Hidden</option>
													</select>
												</div>
											</td>

										</tr>
										
										
										<tr>
											<td>
												<div class="toolbarIcon"  title="Download the current session in xml format"> <img id='saveSessionID'   src="../src/Images/downloadGreen.png" style="padding:5 0; height:25px;"> </div>
											</td>

											<td>
												<div class="toolbarIcon"  title="Upload a session in xml format" > 
													<img id='loadSessionID' class="rotate180" src="../src/Images/downloadGreen.png" style="padding:5 0; height:25px;">
												</div>
											</td>
										</tr>

									</table>
								

									<div style="margin-left: 150px">
										<i>The toolbar contains some essential functions which you should familiarise yourself with.</i> <br><br>

										<b>Reinitiate</b> <img  src="../src/Images/reset.png" style="padding:0 0; width:20px; height:20px"> will reset the polymerase to its initial position. This will
										not affect plots or any other outputs. If you have added a background distribution behind any parameters then they will be resampled. <br><br>

										<b>Stop</b> <img src="../src/Images/stop.png"  style="vertical-align:middle; width:20px; height:20px;"> will immediately halt any polymerase acitivies taking place.
										This will not affect any parameters, plots or other outputs.  <br><br>

										<b>N = </b><input type="number" value=30 style="vertical-align: middle; text-align:left; width: 60px; background-color: #008CBA; color:white; display:inline-block"> is a variable. The meaning of <b>N</b> depends on which
										tool is being used.  <br><br>


										<b>Simulate</b> <img src="../src/Images/dice.png"  style="vertical-align:middle; width:25px; height:25px;"> begins the random simulation starting from the current position
										of the polymerase. During the simulation, reactions are
										randomly sampled according to their rate constants, which can be tweaked in the parameters menu. <b>N</b> trials are performed in the simulation. When the polymerase
										finishes copying the sequence (or prematurely terminates) the next trial begins. The duration of each reaction in the simulation is determined by <b>Speed</b>.
										The simulation can be stopped early using the Stop <img src="../src/Images/stop.png"  style="vertical-align:middle; width:20px; height:20px;"> button.  <br><br>


										<b>Clear cache</b> <img src="../src/Images/close.png"  style="vertical-align:middle; height:20px;"> opens up a dialog where you can delete any kinetic or sequence data which has been saved
										during the simulation. The kinetic data is displayed in plots so think of this button as resetting the plots.  <br><br>


										<b>Copy</b> <img src="../src/Images/transcribe.png"  style="vertical-align:middle; width:25px; height:25px;"> transcribes/replicates/copies the next <b>N</b> bases. The duration of this process is determined by <b>Speed</b>.
										Copying can be stopped early using the Stop <img src="../src/Images/stop.png"  style="vertical-align:middle; width:20px; height:20px;"> button. <br><br>

										<b>Stutter</b> <img src="../src/Images/stutter.png"  style="vertical-align:middle; width:25px; height:25px;"> copies the current base <b>N</b> times, therefore adding an insertion of
										size <b>N</b> into the nascent strand. Insertions are added through slippage. The duration of this process is determined by <b>Speed</b>.
										Stuttering can be stopped early using the Stop </img src="../src/Images/stop.png"  style="vertical-align:middle; width:20px; height:20px;"> button. <br><br>

										<b>Fold</b> <img src="../src/Images/folding.png"  style="vertical-align:middle; width:25px; height:25px;"> folds the mRNA into its predicted secondary structure and renders it onto the nascent strand. This function will only work
										if the nascent strand is single-stranded RNA. Note that this function is only cosmetic and will not affect the polymerase. If you want the mRNA secondary structure to affect reactions
										then make sure that "Enable mRNA folding" is enabled. This function is disabled on mobile devices. <br><br>


										<b>Speed</b> <select class="dropdown" title="Duration of each reaction" style="vertical-align: middle; text-align:right; display:inline-block">
											<option value="slow">Slow</option>
											<option value="medium">Medium</option>
											<option value="fast">Fast</option>
											<option value="ultrafast">Ultrafast</option>
											<option value="hidden">Hidden</option>
										</select>
	
										is a variable which controls the speed of the animation while Copying, Stuttering or Simulating. If the speed is set to <b>slow</b>,
										<b>medium</b> or <b>fast</b>, then the polymerase will animate each reaction at a slow, medium or fast speed, respectively. If the speed is <b>ultrafast</b>,
										then the animation will not show every reaction and will only update periodically. If the speed is <b>hidden</b>, then this is the same as
										<b>ultrafast</b> except that the polymerase is no longer displayed and will not animate at all. <b>Hidden</b> is the fastest setting. <br><br>
										
										<b>Save</b> <img src="../src/Images/downloadGreen.png"  style="vertical-align:middle; height:20px;"> downloads the current session in XML format. If the page is refreshed or if your web browser crashes you will 
										lose your current settings. If you do not want to lose your settings then you should save. <br><br>

										<b>Load</b> <img src="../src/Images/downloadGreen.png" class="rotate180"  style="vertical-align:middle; height:20px;"> uploads a saved XML session file from your local storage.
										 <br><br>


									</div>
								</div>

							</div>
							</div>
					</div>
				</div>


				<br><br><br><br>
				<div class="main-block" >
					<div class="main-block-inner">
						<div class="block-title"> <span id='displayingPlotsBtn' class='minimise' style="font-size:30px; padding: 5 5" disp="true" onClick="minimiseSection(this)">&plus;</span> Plots </div> 
						<div id="displayingPlots"  style="display:none">

						
						
								<div class="about-block">
									<div class="about-block-inner">
										
										
										The output of your simulation can be visualised using various graphs, which are updated in real-time during the simulation.
										These plots are displayed in the Plots panel and in the Sitewise panel. The display of each plot can be configured with its respective settings 
									<img src="../src/Images/gear.png"  style="vertical-align:middle; height:20px;"></img>
									button. The raw data (.tsv) or a high resolution image (.png) can be downloaded <img src="../src/Images/download.png"  style="vertical-align:middle; height:20px;">. The different	
									plots available are detailed below.
							


									</div>
								</div>
							
							
							
								<div  class="about-block">
									<div class="about-block-inner"  style="vertical-align:middle;">
										<div  id="distanceVsTime_PlotHelp" class="block-subtitle"> Distance vs time </div> <br>

										
											<img src="../src/TutorialImages/distanceVsTime.png" style="vertical-align:middle; width:30%; display:block;padding: 10 30;" align=left></img>
											
											<br><br><br><br><br>
											This graph shows the progress made by the polymerase over time and is often used in single-molecule publications. The blue line represents the current simulation
											and the grey lines represent previous simulations. The x- and y-axis ranges can be set manually in the settings <img src="../src/Images/gear.png"  style="vertical-align:middle; height:20px;"></img>.
											

									</div>
								</div>
								
								
								
								<br><br><br>
								<div class="about-block">
									<div class="about-block-inner">
										<div  id="pauseHistogram_PlotHelp"  class="block-subtitle"> Time histogram </div> <br><br>

										
										
											<img src="../src/TutorialImages/pauseHistogram.png" style="vertical-align:middle; width:30%; display:block;padding: 10 30; margin:auto; float:right"></img>
											<img src="../src/TutorialImages/pauseHistogram2.png" style="vertical-align:middle; width:30%; display:block;padding: 10 30; margin:auto; float:right"></img>
											
											<br><br><br>
											Depending on the settings <img src="../src/Images/gear.png"  style="vertical-align:middle; height:20px;"></img>, this will either show the total time taken to copy the template (one value per trial) or the time in
											between successive catalysis events. The x-axis of the latter plot can be restricted so that it only shows pause times (greater than 1s). 
											


									</div>
								</div>
								
								<br><br><br>
								<div class="about-block">
									<div class="about-block-inner" >
										<div  id="velocityHistogram_PlotHelp" class="block-subtitle"> Velocity histogram </div> <br>
											<br>
										
											<img src="../src/TutorialImages/velocityHistogram1s.png" style="vertical-align:middle; width:30%; display:block;padding: 10 30;" align=left></img>
											<img src="../src/TutorialImages/velocityHistogram5s.png" style="vertical-align:middle; width:30%; display:block;padding: 10 30;" align=left></img>
											
											
											
											This is a histogram of the <i>instantaneous</i> velocities during elongation, not the <i> mean </i> velocities. The instantaneous velocity at time $t$ is defined as the derivative of distance over time
											$v(t) = \frac{dx}{dt}$. However since the distance versus time function is discontinuous, there is no derivative. Instead, we calculate the instantaneous velocities as the mean
											velocities within non-overlapping fixed-sized windows. The window size parameter, $w$, can be changed in the settings  <img src="../src/Images/gear.png"  style="vertical-align:middle; height:20px;"></img>.
											The plots on the left illustrate how important this parameter can be. Larger window sizes tend to underrepresent negative velocities, which correspond to net backwards translocation within the time window.
											
											
											


									</div>
								</div>
								
								<br><br><br>
								<div class="about-block">
									<div class="about-block-inner">
										<div id="custom_PlotHelp"  class="block-subtitle"> Parameter plot </div> <br>

										
										<img src="../src/TutorialImages/customPlot2.png" style="vertical-align:middle; width:30%; display:block;padding: 10 30;" align=right></img>
										<img src="../src/TutorialImages/customPlot1.png" style="vertical-align:middle; width:30%; display:block;padding: 10 30;" align=right></img>
										
										<br>
										
										Any parameter in the &#9776; Parameters menu can be plotted onto the x-axis of this graph through the settings menu <img src="../src/Images/gear.png"  style="vertical-align:middle; height:20px;"></img>.
										Note that this parameter will remain constant unless you decide to change it yourself, which you may either do (1) manually, or (2) by adding a prior distribution
										<img src="../src/Images/distn.png" style="vertical-align:middle; width:38px; height:12px;"></img>. By setting the y-axis, you can explore the effect of changing this parameter. Parameters are sampled
										from their prior distribution once per trial. There will be one value added to this plot at the end of each trial. 
										
										

									</div>
								</div>								
								
								<br><br><br>



								<div class="about-block">
									<div class="about-block-inner">
										<div id="parameterHeatmap_PlotHelp"  class="block-subtitle"> Parameter heatmap </div> <br>

										
									
										

									</div>
								</div>								


								<br><br><br>
								<div class="about-block">
									<div class="about-block-inner">
										<div class="block-subtitle"> Dwell time per site </div> <br>



									</div>
								</div>								

						</div>
					</div>		
				</div>
				



				<br><br><br><br>
				<div class="main-block" >
					<div class="main-block-inner">
						<div class="block-title"><span id='parametersBtn' class='minimise' style="font-size:30px; padding: 5 5" disp="true" onClick="minimiseSection(this)">&plus;</span> Parameters </div> 
						<div id="parameters"  style="display:none">
				
		
								
								
								<div class="about-block">
									<div class="about-block-inner">
										
											
											<div style="background-color: #008CBA; width:100%">
											

												<div class="parameter-bg" style="font-size:18px">
													
													<div class="parameter-title" style="font-size:30px">NTP Concentrations </div>  
													<div class="parameter-title" style="font-size:18px">The concentrations of NTP (or dNTP if the nascent strand is DNA). The default values are estimated averages across multiple species and
																have been borrowed from Traut 1994.<cite val="traut1994physiological"></cite> It is recommended that you change these values if you know what they are for your system.</div>  <br> 


														<div class="paramHelp" id="useFourNTPconcentrations_paramHelp">
															<table style="">
															<tr title="Do you want one concentration for all 4 NTPs or one concentration for each one?">

																<td style="text-align:right; width:150px">
														 			1 concentration  
														 		</td>
														 		<td style="" >
															 		<label class="switch">
																 		 <input type="checkbox"> </input>
																 		 <span class="slider round notboolean"></span>
																	</label> 
																	<span style="font-size:15px; vertical-align:middle" >4 concentrations</span>
														 		</td>

													 			<td style="width:75%">
																If this is set to <b>1 concentration</b> then all four NTPs will have the same concentration ([NTP]). If this is set to <b>4 concentrations</b>
																then you can set the concentration of all four NTPs independently ([ATP], [CTP] [GTP], and [UTP]). Default: 1 concentration 

																</td>

															</tr>

														

															</table>
														</div><br>




														<div class="paramHelp" id="NTPconc_ParamHelp">
															<b>[NTP]</b> - concentration of all four NTPs (or dNTPs). Default: 1000&mu;M (or 20&mu;M) 
														</div><br>

														<div class="paramHelp" id="ATPconc_ParamHelp">
															<b>[ATP]</b> - concentration of ATP (or dATP). Default: 3152&mu;M (or 24&mu;M) 
														</div><br>

														<div class="paramHelp" id="CTPconc_ParamHelp">
															<b>[CTP]</b> - concentration of CTP (or dCTP). Default: 278&mu;M (or 29&mu;M) 
														</div><br>

														<div class="paramHelp" id="GTPconc_ParamHelp">
															<b>[GTP]</b> - concentration of GTP (or dGTP). Default: 468&mu;M (or 5.2&mu;M) 
														</div><br>

														<div class="paramHelp" id="UTPconc_ParamHelp">
															<b>[UTP]</b> - concentration of UTP (or dTTP). Default: 567&mu;M (or 37&mu;M) 
														</div><br>
																			   
														
									
												</div>



												<br><br>
												<div class="parameter-bg" style="font-size:18px">
													
													<div class="parameter-title" style="font-size:30px">Structural Parameters </div>  
													<div class="parameter-title" style="font-size:18px">Basepairing configuration within the transcription bubble and polymerase.</div>  <br> 
													


														<div class="paramHelp" id="hybridLen_ParamHelp">
															<b>Hybrid length</b> - number of basepairs in the template/primer hybrid within the polymerase. As translocation rates are determined from sequence composition, this value may affect
																			   the predicted location of pause sites. In eukaryotes and prokaryotes, RNA polymerase contains a hybrid of 8 or 9bp
																			   <cite val="komissarova1997transcriptional,gnatt2001structural,nudler1997rna"></cite>. Default: 9bp 
														</div><br>				   

														<div class="paramHelp" id="bubbleLeft_ParamHelp">			   
															<b>Bubble size 3&prime;</b> - number of unpaired template bases 3&prime; (left hand side) of the hybrid. This parameter is only applicable if the template is double stranded. Default: 2bp<cite val="maoileidigh2011unified"></cite>
														</div><br>	

														<div class="paramHelp" id="bubbleRight_ParamHelp">
															<b>Bubble size 5&prime;</b> - number of unpaired template bases 5&prime; (right hand side) of the hybrid. This parameter is only applicable if the template is double stranded. Default: 1bp<cite val="maoileidigh2011unified"></cite>
														</div><br>	
									
												</div>



												<br><br>
												<div class="parameter-bg" style="font-size:18px">
													
													<div class="parameter-title" style="font-size:30px">Translocation Parameters </div>  
													<div class="parameter-title" style="font-size:18px">Energetics of translocation reactions and forces.</div>  <br> 


														<div class="paramHelp" id="GDagSlide_ParamHelp">
															<b style="font-size:20px">$\Delta G^\dagger_{slide}$</b> - free energy barrier height of translocation. To translocate forwards or backwards, several
																	basepairs may need to be broken. The energy barrier which must be overcome in order to translocate is estimated as the energy needed to break these basepairs
																	plus the value of this parameter. Increasing this value will make translocation slower. Default: 8.7RT 
														</div><br>	

														

														<div class="paramHelp" id="FAssist_ParamHelp">
															<b style="font-size:20px">$F_{assist}$</b> - the size of the force being applied to the polymerase during a single-molecule experiment. Positive forces are assisting loads and increase
																	the rate of transcription. Negative forces are hindering loads and decrease the rate of transcription. When this force reaches a critical value, $k_{cat}$ becomes
																	the rate limiting step. Default: 0pN
														</div><br>	


														<div class="paramHelp" id="arrestTime_ParamHelp">
															<b>Arrest time </b> - the amount of time the polymerase may take to copy a base before the simulation trial is aborted. When the timeout is
																	reached, copying will prematurely terminate and the next trial will begin. Default: 600s 
														</div><br>		
																	

																			   

									
												</div>



												<br><br>
												<div class="parameter-bg" style="font-size:18px">
													
													<div class="parameter-title" style="font-size:30px">NTP Binding Parameters </div>  
													<div class="parameter-title" style="font-size:18px">Kinetics of binding/releasing NTP, and catalysing polymerisation.</div>  <br> 


														<div class="paramHelp" id="RateBind_ParamHelp">
															<b style="font-size:20px">$k_{bind}$</b> - used to calculate rate constant of binding the complementary NTP, which is calculated as [NTP] k<sub>bind</sub>. 
																	See k<sub>misbind</sub> for the rate of binding a non-complementary NTP. This parameter is constant across all NTPs. Default: 30&mu;M<sup>-1</sup>.s<sup>-1</sup>  
														</div><br>


														<div class="paramHelp" id="Kdissociate_ParamHelp">
															<b style="font-size:20px">$K_D$</b> - equilibrium constant of NTP dissociation. $K_D$ is used in conjunction with $k_{bind}$ to calculate the rate of releasing
															bound NTP: $k_\text{release} = K_D \; k_\text{bind}$. $K_D$ is by default shared by all 4 NTPs but you can make it NTP specific instead (ie. 1 term for each of ATP, CTP, GTP, UTP).
															Defaults: $K_D = 35, K_D^{ATP} = 50, K_D^{CTP} = 33, K_D^{GTP} = 36, K_D^{UTP} = 18\;$ <cite val="bai2007mechanochemical"></cite>
														</div><br>


														<div class="paramHelp" id="kCat_ParamHelp"> 
															<b style="font-size:20px">$k_{cat}$</b> - rate constant of catalysing bound NTP, thus extending the nascent chain by one base. When translocation is fast,
																	catalysis is the rate limiting step in transcription. $k_{cat}$ is by default shared by all 4 NTPs but you can make it NTP specific instead (ie. 1 term for each of ATP, CTP, GTP, UTP).
																	Defaults: $k_{cat} = 30, k_{cat}^{ATP} = 38, k_{cat}^{CTP} = 7, k_{cat}^{GTP} = 62, k_{cat}^{UTP} = 24\;$ <cite val="bai2007mechanochemical"></cite>

														</div><br>


														<div class="paramHelp" id="RateMisbind_ParamHelp">
															<b style="font-size:20px">$k_{misbind}$</b> - used to calculate rate constant of binding a non-complementary NTP, which is calculated as [NTP] k<sub>misbind</sub>. 
																		If the bound NTP is added onto the end of the chain without correction then there is a misincorporation event. This parameter is only applicable when "Enable misincorporations" is enabled. The overall rate of uncorrected misincorpations during transcription is
																		around 1 in 10<sup>4</sup> - 10<sup>5</sup> bases<cite val="imashimizu2013direct,rosenberger1981estimate,shaw2002use"></cite>. Default: 0.25s<sup>-1</sup> 
														</div><br>


														<!--
														<div class="paramHelp" stule="display:none">
															<b style="font-size:20px">$k_{transition} / k_{transversion}$</b> - the ratio between transition and transversion rates. This parameter is only applicable when "Allow inactivation" is enabled. Default: 2 
														</div><br>
														-->
														
																			   

									
												</div>


												<br><br>
												<div class="parameter-bg" style="font-size:18px">
													
													<div class="parameter-title" style="font-size:30px">Unactivated State Parameters </div>  
													<div class="parameter-title" style="font-size:18px">Kinetics of entering/exiting a catalytically inactive state.</div>  <br> 


														<div class="paramHelp" id="kU_ParamHelp">
															<b style="font-size:20px">$k_{U}$</b> - rate constant of polymerase entering the unactive state. When in the unactive state, the polymerase may
																	translocate but it may not bind NTP or catalyse. Elongation may only continue when the polymerase reactivates. Default: 0.23s<sup>-1</sup> <cite val="maoileidigh2011unified"></cite>
														</div><br>

														<div class="paramHelp" id="kA_ParamHelp">
															<b style="font-size:20px">$k_{A}$</b> - rate constant of polymerase entering the active state. Default: 3s<sup>-1</sup> <cite val="maoileidigh2011unified"></cite>
														</div><br>
												</div>




												<br><br>
												<div class="parameter-bg" style="font-size:18px">
													
													<div class="parameter-title" style="font-size:30px">RNA Folding Parameters  </div>  
													<div class="parameter-title" style="font-size:18px">Settings behind folding the nascent RNA strand.</div>  <br> 
													
														<b>Enable mRNA folding </b> - whether or not the nascent RNA strand will fold automatically during the simulation. The predicted energy barrier 
																of unfolding secondary structures in the mRNA may need to be overcome when the polymerase translocates backwards. When the nascent strand
																is DNA or dsRNA this feature is not applicable. Note that enabling this feature will slow down the first trial of the simulation as we compute
																the predicted (MFE<cite val="zuker1981optimal"></val></cite>) secondary structure of the RNA. You can view the RNA secondary structure with the Fold 
																<img src="../src/Images/folding.png"  style="vertical-align:middle; width:25px; height:25px;"></img> button. Default: TRUE <br><br>


														<div class="paramHelp" id="nbpToFold_ParamHelp">
															<b>Number of bases to fold </b> - the number of most recently added bases in the nascent strand to fold (if "Allow mRNA folding" is enabled). The timescale
																of transcription is probably not long enough to allow the whole mRNA chain to fold<cite val="maoileidigh2011unified"></cite>, especially in vivo, and folding the entire chain is
																computationally expensive. For these two reasons we do not fold the entire mRNA but only the most recently added bases. Default: 150nt<cite val="maoileidigh2011unified"></cite> <br><br>
														</div><br>
												</div>




												<br><br>
											
										
											
											</div>
											
										
									</div>
								</div>



								



								
								
								<br><br>
								<div class="about-block">
									<div class="about-block-inner">
										<div id="priorDistribution_ParamHelp" class="block-subtitle" > Adding a background distribution to a parameter</div> <br>
											
											

											
											
											<br>
											If you are uncertain about the value of a parameter you can give it a background distribution <img src="../src/Images/distn.png" style="vertical-align:middle; width:38px; height:12px;"></img> 
											to capture your uncertainty. The probability distributions supported are uniform, normal, lognormal, exponential, and gamma. The parameter is resampled at the beginning of each simulation. 
											All parameters are sampled independently of each other.
											<br><br>
											
											
											<b>Example 1:</b> [ATP] is defaulted to Traut 1994â€™s<cite val="traut1994physiological"></cite> estimate of the mean ATP concentration (3152&mu;M) but one may wish to refine this value to reflect their own 
											experimental settings. Or one may recognise that cellular NTP concentrations are not constant, and they fluctuate throughout the cell cycle. In this case, it may be more accurate to 
											add a prior distribution behind [ATP]. Yaginuma et al. 2014 <cite val="yaginuma2014diversity"></cite> describe heterogeneity in [ATP] across E.coli cells with [ATP] following a positive-skewed modal distribution 
											(mean: 1540&mu;M, SD: 1220&mu;M). Using SimPol and some algebra, one could model [ATP] approximately with a lognormal distribution (where &mu; = 7.10, &sigma; = 0.698). Then the ATP 
											concentration would be resampled from this distribution at the beginning of each trial. <br><br>
											
											<!--
											<b>Example 2:</b> There can be a great deal of heterogeneity between different RNA polymerase units. The standard deviation of transcription velocity 
											across different RNA polymerase units in E.coli is quite high: around 4-6bp/s<cite val="adelman2002single,tolic2004diversity"></cite>. Compare this to the mean velocity of 
											14-20bp/s. One of many methods to account for polymerase heterogeneity is to add a normal prior distribution behind $k_{cat}$ with a standard deviation of, for example, 5.<cite val="maoileidigh2011unified"></cite>
											-->
											


											

									</div>
								</div>
								
								

											

								<br><br><br>


						</div>
					</div>		
				</div>





				<br><br><br><br>
				<div class="main-block" >
					<div class="main-block-inner">
						<div class="block-title"><span id='kineticsBtn' class='minimise' style="font-size:30px; padding: 5 5" disp="true" onClick="minimiseSection(this)">&plus;</span>  Kinetics and Thermodynamics </div> 
						<div id="kinetics"  style="display:none">


						
						
							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div class="block-subtitle" > Equilibrium and the Boltzmann distribution </div> <br>
								
										Gibb's free energy, $\Delta G$, describes how energetically favourable a state is. A system will spend more time in states which have low free energies. 
										Suppose we have an ensemble of molecules and each molecule can exist in a number of states. When the system has reached equilibrium, the probability of the system being in state $s$ 
										at any given time is estimated by a Boltzmann distribution:
										
										<h3>
										$$ p(s) \propto e^{-\frac{\Delta G_s}{k_B T}}, $$
										</h3>
										
										where $\Delta G_s$ is the free energy of state $s$, $k_B$ is Boltzmann's constant ($1.3806 \times 10^{-23}$J.K$^{-1}$), and $T$ is the temperature in Kelvins. 
										We express free energy in units of $k_B T$, by taking the experimental energy (in units of J) and dividing by $k_B T$.   <br><br>
										
										For example, suppose a single molecule is in equilibrium between two states, $A$ and $B$. These two states have free energies of $\Delta G_A = -3k_B T$ and $\Delta G_B = -6k_B T$,
										respectively. We would therefore expect the molecule to be in state $A$ 4.7% of the time:
										
										
										<h3>
											$$p(A) = \frac{e^{-\frac{\Delta G_A}{k_B T}}}{e^{-\frac{\Delta G_A}{k_B T}} + e^{-\frac{\Delta G_B}{k_B T}}} = 
													 \frac{e^{-\frac{-3k_B T}{k_B T}}}{e^{-\frac{-3k_B T}{k_B T}} + e^{-\frac{-6k_B T}{k_B T}}}  = 
													 \frac{e^{3}}{e^{3} + e^{6}}  =
													 0.047.$$
										</h3>
										
			
										
										<br>
										The Boltzmann distribution assumes that the system has been fluctuating for long enough to achieve equilibrium. However transcription occurs on too fast a timescale for equilibrium to be reached. 
										Therefore we must model transcription as a kinetic process instead of an equilibrium one.
							
								</div>
							</div>
							
							
							
							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div class="block-subtitle" > Kinetics and the Arrhenius equation </div> <br>
								
										
										Chemical kinetics is the study of rates of chemical processes, and how long reactions take to occur. A chemical reaction can be described with a rate constant $k$ (units s$^{-1}$). 
										$k$ describes the speed of the reaction - a higher rate means that more reactions are expected to occur in a given time. 
										
										
										
										Energy is required for a system to transit from one state to another. 
										This free energy barrier is denoted by $\Delta G^\dagger$. The larger the barrier, the more time needed for random thermal collisions to provide
										enough energy to allow the system to make the transition. Suppose a system is currently in state $s$, with free energy $\Delta G_s$. 
										The Arrhenius equation can estimate the rate constant of transiting from state $s$ to state $s^\prime$:
										
										
										<h3>
											$$ k_{s \rightarrow s^\prime} = A e^{-\frac{\Delta G^\dagger - \Delta G_s}{k_B T}}, $$
										</h3>
										
										where $A$ is a pre-exponential constant (units s$^{-1}$), and $\Delta G^\dagger$ is the absolute height of the free energy barrier between states $s$ and $s^\prime$. 
										Once the rate constants of all possible reactions are known, we can start the single-molecule simulation. 
										
										
										
							
								</div>
							</div>
							
							
							
							
							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div class="block-subtitle"> Poisson processes: how a reaction is randomly sampled </div> <br>
								
									A critical assumption is that these reactions are Poisson processes; the amount of time until a reaction occurs is independent of all other reactions and independent of how long you have already waited. 
									If you were to keep flipping a coin until it landed on tails, the probability of getting tails on the next flip is not affected by how many times you have already flipped, and is not affected by how 
									many times your friend has flipped their coin. Similarly, whether you have been waiting 1 second or 1 hour for a chemical reaction to occur, the probability of it happening right now is always the same.
									The amount of time until reaction $s \rightarrow s^\prime$ occurs follows the exponential distribution with rate $k$. <br><br>
									
									<img id="exponential" src="../src/TutorialImages/exponential.png" style="height:350px; padding: 10 30;" align=left></img>
									
									If there are multiple chemical reactions competing with each other, then the reaction with the shortest reaction time will be selected. For example, suppose that you have a test-tube containing 
									a single molecule of compound $A$, which can spontaneously undergo two reactions: $A \rightarrow B$ or $A \rightarrow C$, with rate constants 0.001s<sup>-1</sup> and 0.002s<sup>-1</sup> respectively. 
									The test-tube will flash a different colour of light for each reaction. <br><br>
									
									
									The amount of time until either one of these reactions occurs is random, and exponentially distributed (see figures on the left). If we were to simulate this on a computer, we could 
									generate two exponential random variables, one from each distribution. For example, suppose we sampled time 769.14s for Exponential(0.001) and 339.81s for Exponential(0.002). Then 
									reaction $A \rightarrow C$ would be selected because it has the shorter reaction time. The probability of sampling reaction $A \rightarrow B$ is 
									
									
									<h3>
										$$ p(A \rightarrow B) = \frac{0.001}{0.001 + 0.002} = 0.333,$$
									</h3>
										
									
									
									and the probability of sampling reaction $A \rightarrow C$ is $0.667$. 
									
									Reactions are sampled this way when you press the Simulate <img src="../src/Images/dice.png"  style="vertical-align:middle; width:25px; height:25px;"></img> button.
									
									
									
									
							
								</div>
							</div>
				

						
						
							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div id="polymerisation_ModelHelp" class="block-subtitle" > Kinetics of nucleotide incorporation</div> <br>

										
										Adding the next base onto the $3^\prime$ end of the nascent strand is a four step process <cite val="erie1992single"></cite>.
										
										
										
										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b>1. NTP binding.</b> A free nucleoside triphosphate (NTP) molecule enters the active site and binds to the active site and its complementary base in the template. 

										</div><br>
										
										
										
										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b>2. Triphosphate cleavage.</b> The triphosphate in the bound NTP is cleaved, resulting in a nucleoside monophosphate (NMP) and a pyrophosphate, both still bound within the active site.

										</div><br>
										
										
										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b>3. Phosphodisester bond formation.</b> The $5^\prime$ phosphate of the NMP is linked to the $3^\prime$-OH of the nascent strand.

										</div><br>
										
		

										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b>4. Pyrophosphate release.</b> The bound pyrophosphate is released back into solution.

										</div><br><br>	
										
										Any of these steps could be reversible.
										As there is not sufficent information to model all four steps, nucleotide incorporation has been compressed into to a two step process.
										
										
										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b>1. NTP binding.</b> Step 1 from above. This is a reversible process. NTP binding/release is dictated by two parameters: $k_{bind}$ and $K_D$. 
		

										</div><br>
										
										
										
										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b>2. Catalysis.</b> Steps 2-4 from above. This is assumed to be an irreversible process. The rate of incorporating a bound nucleotide is $k_{cat}$.

										</div><br><br>	
										
										
										 
											$k_{bind}$ (units s$^{-1}$&mu;M$^{-1}$) is the second order rate constant of binding NTP.
											The rate of binding an NTP is equal to $k_{bind}$[NTP]. Once the NTP has been bound, it may be released back into solution.
											The rate of release is calculated from the dissociation constant $K_D$ (units &mu;M). Rate of NTP release is equal to $K_D \times k_{bind}$. 
											<br><br>
											
											
											When the NTP concentration is high, or if $k_{bind}$ is large, then NTP binding becomes so rapid that equilibrium is achieved. Under these conditions,
											the kinetics of NTP binding are no longer relevant. $K_D$ simply describes the proportion of time that NTP is bound. For example, if $K_D = 35$&mu;M and [NTP] = 1000&mu;M,
											then the proportion of time that NTP is bound (while in this two-state system) is around 97%:
											
										
											
											<h3>
												$$ p(\text{bound}) = \frac{ \frac{\text{[NTP]}}{K_D} }{\frac{\text{[NTP]}}{K_D} + 1} = \frac{ \frac{1000\text{&mu;M}}{35 \text{&mu;M}} }{\frac{1000\text{&mu;M}}{35 \text{&mu;M}} + 1} = 0.966, $$
											</h3>
										
											<br>
											$k_{cat}$ is the rate of incorporating a bound NTP. This reaction is one of the rate limiting steps of transcription. <br><br>
											
											
											<span style="line-height:28px;">
												In SimPol, by default there is one concentration for all four NTPs, and one dissociation constant and rate of catalysis for all four bases. You can set the concentrations
												and incorporation parameters for each of the four nucleotides independently using 
												
												<span style="background-color:#ebe9e7; padding: 7 6;">
													<span style="font-size:15px; vertical-align:middle; font-family:Arial" >1 concentration</span>
													<label class="switch">
														 <input type="checkbox"> </input>
														 <span class="slider round notboolean"></span>
													</label> 
													<span style="font-size:15px; vertical-align:middle; font-family:Arial" >4 concentrations</span>
												</span>
												
												&nbsp;and 
												
												<span style="background-color:#ebe9e7; padding: 7 6;">
													<span style="font-size:15px; vertical-align:middle; font-family:Arial" >2 params</span>	
													<label class="switch">
														 <input type="checkbox"> </input>
														 <span class="slider round notboolean"></span>
													</label> 
													<span style="font-size:15px; vertical-align:middle; font-family:Arial" >8 params</span>	
												</span>											
												
												&nbsp;respectively. While $K_D$ and $k_{cat}$ can be set independently for each nucleotide, $k_{bind}$ is held constant for all four.
											
											</span>
											
											
								
							
								</div>
							</div>


						
						
						
							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div id="translocation_ModelHelp" class="block-subtitle" > Calculating translocation rates</div> <br>
								
										Basepairs may need to be broken in order for the polymerase to translocate forwards (right) or backwards (left) from its current position. For example, consider the state below. 

										<br><br>


										<img id="breakingPairs" src="../src/TutorialImages/breakingPairs.png" style="width:1200px; display:block;  margin: 0 auto;" border="1" ></img>

										<br><br>

										<div style="line-height:35px">
											For the polymerase to move one step to the left, the 
										
											<span style="position:absolute; width:33px; "> 
												<img src="../src/Images/Ag.png" style="width:15px; height:15px; position:absolute; left:0px"></img>
												<img src="../src/Images/Gg.png" style="width:15px; height:15px; position:absolute; left:18px"></img>
												<img src="../src/Images/Tm.png" style="width:15px; height:15px; top:18px; position:absolute; left:0px"></img>
												<img src="../src/Images/Cm.png" style="width:15px; height:15px; top:18px; position:absolute; left:18px"></img>
											</span> 



											<span style="width:33;  display:inline-block;"></span> doublet on the left of the transcription bubble and the 



											<span style="position:absolute; width:33px; "> 
												<img src="../src/Images/Ag.png" style="width:15px; height:15px; position:absolute; left:0px"></img>
												<img src="../src/Images/Cg.png" style="width:15px; height:15px; position:absolute; left:18px"></img>
												<img src="../src/Images/Um.png" style="width:15px; height:15px; top:18px; position:absolute; left:0px"></img>
												<img src="../src/Images/Gm.png" style="width:15px; height:15px; top:18px; position:absolute; left:18px"></img>
											</span> 


											<span style="width:33;  display:inline-block;"></span> doublet on the right of the hybrid must both be broken. Additionally, the mRNA must be refolded into a different conformation. For the polymerase
											to move one step to the right, the 




											<span style="position:absolute; width:33px; "> 
												<img src="../src/Images/Gg.png" style="width:15px; height:15px; position:absolute; left:0px"></img>
												<img src="../src/Images/Tg.png" style="width:15px; height:15px; position:absolute; left:18px"></img>
												<img src="../src/Images/Cm.png" style="width:15px; height:15px; top:18px; position:absolute; left:0px"></img>
												<img src="../src/Images/Am.png" style="width:15px; height:15px; top:18px; position:absolute; left:18px"></img>
											</span> 


											<span style="width:33;  display:inline-block;"></span> doublet on the right of the transcription bubble and the 



											<span style="position:absolute; width:33px; "> 
												<img src="../src/Images/Ag.png" style="width:15px; height:15px; position:absolute; left:0px"></img>
												<img src="../src/Images/Ag.png" style="width:15px; height:15px; position:absolute; left:18px"></img>
												<img src="../src/Images/Um.png" style="width:15px; height:15px; top:18px; position:absolute; left:0px"></img>
												<img src="../src/Images/Um.png" style="width:15px; height:15px; top:18px; position:absolute; left:18px"></img>
											</span> 


											<span style="width:33;  display:inline-block;"></span> doublet on the left of the hybrid must both be broken. 

										</div>

										The estimated basepairing contribution to free energy for any given state can readily be calculated using RNA or DNA free energy parameters. This program uses these values to estimate the free energy 
										barrier heights of translocating forwards and backwards. Let $\Delta G_x$ and $\Delta G_y$ be the basepairing free energies of neighbouring states x and y, respectively. Then, the free energy barrier
										height of transition from state x to y is calculated as 
										
										$$
											\Delta G^\dagger_{x,y} = \frac{\Delta G_x + \Delta G_y}{2} + \Delta G^\dagger_{slide},
										$$
										
									
										
										where the value of parameter $\Delta G^\dagger_{slide}$ can be manually tweaked. This is a very simplistic estimation of translocation barrier heights. 
										The rate constants of translocation are derived from the free energy barrier heights using the Arrhenius equation with a pre-exponential constant of $10^6$ s<sup>-1</sup>:

										
										$$
											k_{x \rightarrow y} = 10^6 e^{-(\Delta G_{x,y}^\dagger - \Delta G_x)}.
										$$
										
				
										This model of translocation has been borrowed from Tadigotla et al.<cite val="maoileidigh2011unified,tadigotla2006thermodynamic"></cite>


								
							
								</div>
							</div>




						



					
						</div>
					</div>
				</div>



				<br><br><br><br>
				<div class="main-block">
					<div class="main-block-inner">
						<div class="block-title"> 
						
						
							<span id='ABCSectionBtn' class='minimise' value='&minus;' style="font-size:30px; padding: 5 5" disp="true" onClick="minimiseSection(this)">&plus;</span>
							Approximate Bayesian Computation
						</div> 
						<div id="ABCSection"  style="display:none">


							<br><br>
								<div class="about-block">
									<div class="about-block-inner">
									<div id="setupABC_ABCSectionHelp" class="block-subtitle" >Setting up ABC</div> <br>

										Approximate Bayesian Computation (ABC) is a simple statistical inference method, which SimPol uses to fit parameters to experimental data. 


									</div>
								</div>

						</div>
					</div>
				</div>

	
				
				<br><br><br><br>
				<div class="main-block">
					<div class="main-block-inner">
						<div class="block-title"> 
						
						
							<span id='performanceBtn' class='minimise' value='&minus;' style="font-size:30px; padding: 5 5" disp="true" onClick="minimiseSection(this)">&plus;</span>
							Improving Software Performance 
						</div> 
						<div id="performance"  style="display:none">
				
				

							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div class="block-subtitle" > Web browser performance </div> <br>
										
										Rendering elements onto the screen (for example polymerase motion, plots, simulated sequences) can be computationally costly and can slow down the whole program.  If you are running a complex simulation on a 
										long sequence over a large number of trials, you may notice the simulation begin to slow down eventually. Maximising the performance of this program requires you as the user to regulate when rendering does or 
										does not occur. If the program starts to slow down, you should hide components of the interface with its respective minimise <span class='minimise' value='-' style="cursor:auto; padding: 1 1">&minus;</span>
										button. If a component is minimised then rendering will not occur. The polymerase animation can be minimised by setting the speed to <b>hidden</b>. This will happen automatically when simulating large sequences. <br><br>


										Your web browser may attempt to divert resources away from your session if you are running for a long time. If this is a problem,
										you should consider downloading SimPol and running it through the command line using Node.js. Node.js employs the powerful V8 JavaScript engine
										used by Chrome.  <br><br>


										In October 2017 we tested SimPol on different operating systems using recent versions of different web browsers, and using Node.js. 
										Each setting was tested on 1000 simulations of the 4029nt E.coli ropB gene (accession number KP670789). 

										Chrome was the fastest web browser on all devices we tested.
										
										Stripping away the user interface and running with Node.js gives performance only slightly faster than the Chrome browser. Therefore the user interface of SimPol 
										does not slow down the simulation to any significant degree.
										
										<span style="padding: 1 5; background-color:#ebe9e7"> 
											<b style="color:#008cba">Benchmark session:</b> 
											<a style="text-decoration:none" href="../?benchmark.xml">Open</a> | 
											<a style="text-decoration:none" href="Examples/benchmark.xml">View XML</a>
										</span>										


										<br> <br>


										
										<table cellpadding=5 style="margin:auto; text-align:center;">
											<tr style="background-color:#b3b3b3; font-size:18px">
												<td>
													OS
												</td>
												<td>
													Device description
												</td>
												<td>
													Program
												</td>
												<td>
													Time (min)
												</td>
											</tr>
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td rowspan=3  style="background-color:#E95420">
													Ubuntu 7
												</td>
												
												<td rowspan=3>
													Intel i5 3.30GHz
												</td>
												
											
												<td >
													<b>Node 8.6.0</b>
												</td>
												
												<td>
													1:55
												</td>
												
											</tr>

											<tr style="background-color:#ebe9e7; font-size:18px">
												
												
												<td >
													<b>Chrome 61</b>
												</td>
												
												<td>
													1:55
												</td>
												
											</tr>
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
											
												<td >
													<b>Firefox 56</b>
												</td>
												
												<td>
													3:31
												</td>
												
											</tr>
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td rowspan=4  style="background-color:#00a1f1">
													Windows 10
												</td>
												
												<td rowspan=4>
													Intel i7 2.80GHz
												</td>
												
											
												<td >
													<b>Node 8.6.0</b>
												</td>
												
												<td>
													3:19
												</td>
												
											</tr>
											
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
											
											
												<td >
													<b>Chrome 61</b>
												</td>
												
												<td>
													3:27
												</td>
												
											</tr>
											
											
																						
											<tr style="background-color:#ebe9e7; font-size:18px">
												
											
											
												<td >
													<b>Firefox 56</b>
												</td>
												
												<td>
													6:30
												</td>
												
											</tr>
											
											
											
																						
											<tr style="background-color:#ebe9e7; font-size:18px">
												
											
											
												<td >
													<b>Microsoft Edge 40</b>
												</td>
												
												<td>
													14:31
												</td>
												
											</tr>
											
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td rowspan=2  style="background-color:#999999">
													OS X 10.11
												</td>
												
												<td rowspan=2>
													Intel i5 2.70GHz
												</td>
												
											
												<td >
													<b>Chrome 61</b>
												</td>
												
												<td>
													
												</td>
												
											</tr>
		
											
											
												
											<tr style="background-color:#ebe9e7; font-size:18px">
												

												<td >
													<b>Safari 10.1</b>
												</td>
												
												<td>
													
												</td>
												
											</tr>
											
											
												
											<tr style="background-color:#ebe9e7; font-size:18px">

												<td rowspan=1  style="background-color:#999999">
													iOS 10.0.3
												</td>
												
												<td rowspan=1>
													iPhone 7
												</td>

												<td >
													<b>Safari 10.3</b>
												</td>
												
												
												<td>
													2:59
												</td>
												
											</tr>

											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td rowspan=2  style="background-color:#a4c639">
													Android 7.0
												</td>
												
												<td rowspan=2>
													Samsung Galaxy S6 Edge+
												</td>
												
											
												<td >
													<b>Chrome 61</b>
												</td>
												
												<td>
													6:53
												</td>
												
											</tr>
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												
											
												<td >
													<b>Firefox 56</b>
												</td>
												
												<td>
													13:13
												</td>
												
											</tr>
											
											
											
										


											
											
										
								</table>

	


								</div>
							</div>



							<!--
							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div class="block-subtitle"> Web workers </div> <br>
										
										Setting the simulation speed to <b>slow</b>, <b>medium</b> or <b>fast</b> will determine the animation time of each reaction.
										If you don't need to see the animations then you can set the speed to <b>ultrafast</b> or <b>hidden</b>. In these modes,
										the operations all occur behind the scenes and the HTML will only update periodically or not at all. Sampling reactions is resource intensive 
										and can make the browser unresponsive. For this reason we try to run these background processes in a Web Worker while rendering the animation is done in the main window. <br><br>
										
										A Web Worker is like a socket, or an invisible tab your browser opens which runs on its own thread and can communicate
										with the main window. <a href="https://www.w3schools.com/html/html5_webworkers.asp">Most modern browsers support Web Workers</a> but if 
										your browser is not up to date or if you have an overzealous third-party plugin then the Web Worker may not register (SimPol will notify you with an error message). 
										We recommend that you allow the Web Worker to register, to ensure the whole program runs smoothly and you can run simulations in <b>ultrafast</b> or <b>hidden</b> mode.

								</div>
							</div>
							-->




							<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div class="block-subtitle"> Running SimPol from the command line </div> <br>
										

										If you do not wish to run SimPol through a web browser, or if you plan to use a cluster then you should download SimPol and run it 
										through the command line.  <br> <br>


										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b> 1. Download SimPol </b> from <a href="../download">here</a> and unzip.

										</div> <br>


										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b> 2. Install Node.js >= 8.6.0.</b> Follow the instructions at the <a href="https://nodejs.org/en/download/current/">Node.js website</a>.  

										</div> <br>


										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b> 3. Generate a SimPol session file.</b> Open up SimPol in the Web browser and configure the number of trials, parameters, model settings, ABC settings, etc.
											When you are happy with all the settings, download the session as an XML file <img id='saveSessionID'   src="../src/Images/downloadGreen.png" style="padding:5 0; vertical-align:middle; height:25px;">.

										</div> <br>


										<div style="padding: 10 10; background-color:#ebe9e7" >

											<b> 4. Run SimPol using Node.js.</b> Open up the command terminal. Navigate to the folder which contains simpol.js and enter: <br><br>

												<div class="code" style="margin-left: 100px;">
													
													node simpol.js session.xml

												</div> <br>

												where <span class="code">session.xml</span> is the session you generated earlier. This will run the number of simulations
												you specified and by default it will not save any kinetic data. Optional arguments:<br><br>

												<div style="margin-left: 100px;">
													
													<span class="code">-ABC</span> : Run ABC analysis instead of just simulating. <br><br>

													<span class="code">-o outputfolder</span> : Write the output into files contained within the specified folder. If the folder does not exist
													it will be generated. If running simulations, distance versus time, time per site, and parameter data will be saved into
													this folder. If <span class="code">-ABC</span> is enabled, then the posterior distribution will be saved into this folder instead.

												</div>


										</div>


								</div>
							</div>
				
				
						</div>
					</div>
				</div>



	<!--
	<br><br><br><br>
	<div class="main-block">
		<div class="main-block-inner">
			<div class="block-title"> 
			
			
				<span id='examplesBtn' class='minimise' value='&minus;' style="font-size:30px; padding: 5 5" disp="true" onClick="minimiseSection(this)">&plus;</span>
				Examples 
				</div> 
					<div id="examples"  style="display:none">

						<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									<div class="block-subtitle" > Example 1: effect of hybrid size on pause sites </div> <br>


									Open up the session <a href="../?test.xml">test.xml</a>. 



								</div>
							</div>


					</div>
				</div>
	
	
			</div>
		</div>
	</div>
	-->
	
	
	
	<br><br><br><br>
	<div class="main-block">
		<div class="main-block-inner">
			<div class="block-title"> 
			
			
				<span id='dataBtn' class='minimise' value='&minus;' style="font-size:30px; padding: 5 5" disp="true" onClick="minimiseSection(this)">&plus;</span>
				Kinetic Data 
				</div> 
					<div id="data"  style="display:none">

						<br><br>
							<div class="about-block">
								<div class="about-block-inner">
									


									Transcription elongation velocities (bp/s) below. <br><br>
									
									
									
										<table cellpadding=5 style="margin:auto; text-align:center; vertical-align:middle">
											<tr style="font-size:18px">
												<td>
													
												</td>
												<td  style="background-color:#b3b3b3;">
													RNA polymerase
												</td>
												<td style="background-color:#b3b3b3;">
													Optical trapping
												</td>
												<td style="background-color:#b3b3b3;">
													Other technique
												</td>
											</tr>
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												<td rowspan=1 style="background-color:#b3b3b3;">
													Prokaryote
												</td>
												<td>
													<i>Escherichia coli</i> RNAP
												</td>
												
												<td>
													7.3 bp/s <cite val="davenport2000single"></cite> <br>
													9.4 bp/s <cite val="neuman2003ubiquitous"></cite> <br>
													11.7 bp/s <cite val="mejia2015trigger"></cite> <br>
													12.8 bp/s <cite val="adelman2002single"></cite> <br>
													15.9 bp/s <cite val="tolic2004diversity"></cite> <br>
													20.6 bp/s <cite val="bai2007mechanochemical"></cite> <br>
													23 bp/s <cite val="abbondanzieri2005direct"></cite>
												</td>
												
												<td>
													25 bp/s <cite val="iost1992bacteriophage"></cite> <br>
													28-89 bp/s <cite val="vogel1994rna"></cite> <br>
													30-103 bp/s <cite val="ryals1982temperature"></cite> <br>
													39-85 bp/s <cite val="bremer1996modulation"></cite>  <br>
													
													90 bp/s <cite val="richardson1996control"></cite> <br>
													
													
													
													
												</td>
												
											</tr>
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												<td rowspan=5 style="background-color:#b3b3b3;">
													Eukaryote
												</td>
												<td>
													<i>Human</i> pol II
												</td>
												
												<td>
													
													
												</td>
												
												<td>
												
												
													<table style="width:100%; text-align:center;  font-size:18px;" cellpadding=5>
												
														<tr>
														
															<td>
																<i> FRAP </i> <br>
																30 bp/s <cite val="boireau2007transcriptional"></cite> <br>
																72 bp/s <cite val="darzacq2007vivo"></cite> <br><br>
															</td>
															
															<td>
																<i> qRT-PCR </i> <br>
																40 bp/s <cite val="tennyson1995human"></cite> <br>
																63 bp/s <cite val="singh2009rates"></cite> <br><br>
															</td>
															
														</tr>
														
														
															<tr>
														
															<td>
																
																50-100 bp/s <cite val="sehgal19765"></cite> <br>
															</td>
															
															<td>

															</td>
															
														</tr>
													
													</table>
													
													
													
												</td>
												
											</tr>
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td>
													<i>Saccharomyces cerevisiae</i> pol I
												</td>
												
												<td>
													
													
												</td>
												
												<td>
												
													40 bp/s <cite val="kovs2010yeast"></cite> <br>
													60 bp/s <cite val="french2003exponentially"></cite> <br>
													
													
													
												</td>
												
											</tr>
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td>
													<i>Saccharomyces cerevisiae</i> pol II
												</td>
												
												<td>
													10-20 bp/s <cite val="galburt2009single"></cite> <br>
													17.5 bp/s <cite val="hodges2009nucleosomal"></cite> <br>
													25 bp/s <cite val="dangkulwanich2013complete"></cite> <br>
													
													
													
												</td>
												
												<td>
												
													33.3 bp/s <cite val="mason2005distinction"></cite> <br>
													
													
													
													
												</td>
												
											</tr>
											
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td>
													<i>Rat</i> pol II
												</td>
												
												<td>
													
													
												</td>
												
												<td>
												
													20-25 bp/s <cite val="ucker1984early"></cite> <br>
													22 bp/s <cite val="mejia2015trigger"></cite> <br>
													
													
												</td>
												
											</tr>


											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td>
													<i>Drosophila</i> pol II
												</td>
												
												<td>
													
													
												</td>
												
												<td>
												
													18 bp/s <cite val="thummel1990spatial"></cite> <br>
													22 bp/s <cite val="o1993rapid"></cite> <br>
													23 bp/s <cite val="shermoen1991progression"></cite> <br>
													25 bp/s <cite val="yao2007intranuclear"></cite> <br>
													
													
												</td>
												
											</tr>
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												<td rowspan=1 style="background-color:#b3b3b3;">
													dsDNA virus
												</td>
												<td>
													<i>Bacteriophage T7</i> pol
												</td>
												
												<td>
													43 bp/s <cite val="skinner2004promoter"></cite> <br>
													98-141 bp/s <cite val="thomen2008t7"></cite> <br>
													
												</td>
												
												<td>
													200 bp/s <cite val="iost1992bacteriophage"></cite> <br>
													222 bp/s <cite val="anand2006transient"></cite> <br>
													240 bp/s <cite val="makarova1995transcribing"></cite> <br>
													
			
												</td>
												
											</tr>
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												<td rowspan=1 style="background-color:#b3b3b3;">
													dsRNA virus
												</td>
												<td>
													<i>Bacteriophage &Phi;6</i> P2
												</td>
												
												<td>
													16.7 bp/s <cite val="dulin2015elongation"></cite> <br>
													
												</td>
												
												<td>
													20-30 bp/s <cite val="kainov2004packaging"></cite> <br>
													
			
												</td>
												
											</tr>
											
											
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												<td rowspan=3 style="background-color:#b3b3b3;">
													(-)ssRNA virus
												</td>
												<td>
													<i>Sendai virus</i> L
												</td>
												
												<td>
													
													
												</td>
												
												<td>
													1.7 bp/s <cite val="gubbay2001sendai"></cite> <br>
													
			
												</td>
												
											</tr>
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td>
													<i>Measles virus</i> L
												</td>
												
												<td>
													
													
												</td>
												
												<td>
													3 bp/s <cite val="plumet2005dynamics"></cite> <br>
													
			
												</td>
												
											</tr>
											
											
											<tr style="background-color:#ebe9e7; font-size:18px">
												
												<td>
													<i>Vesicular stomatitis virus</i> L
												</td>
												
												<td>
													
													
												</td>
												
												<td>
													3.1-4.3 bp/s <cite val="iverson1981localized"></cite> <br>
													
			
												</td>
												
											</tr>

										

											
											
										
								</table>



								</div>
							</div>


					</div>
				</div>
	
	
			</div>
		</div>
	</div>


	
	
	
	<br><br><br><br>
	<div class="main-block">
		<div class="main-block-inner">
			<div class="block-title"> 
			
			
				<span id='referencesSectionBtn' class='minimise' value='&minus;' style="font-size:30px; padding: 5 5" disp="true" onClick="minimiseSection(this)">&plus;</span>
				References 
				</div> 
					<div id="referencesSection" style="display:none">

						<br><br>
						<div class="about-block" style="width:90%; margin:auto">
							<div class="about-block-inner"  id="references">
									
									
									

							</div>
						</div>


					</div>
				</div>
	
	
			</div>
		</div>
	</div>
	

	
	
	
	
	<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br>
	<br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> <br> 

	
	
		<div class="header">
			SimPol, Lab of Structural Biology & Centre for Computational Evolution, University of Auckland
		</div>
	


		</div>
	</div>


</div>
</body>
</html>
